code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -575,6 +640,8 @@
 	activate_reagents = list(/datum/reagent/blood,/datum/reagent/toxin/plasma,/datum/reagent/water)
 
 /obj/item/slime_extract/sepia/activate(mob/living/carbon/human/user, datum/species/species, activation_type)
+	if(activating)
```

Comment:
Wait a minute instead of doing this once for every slime extract, why not just do it once on the parent

Reply:
True. I will do that.

Reply:
Ah, I see now. The parent isn't called on the subtypes. So I could add a parent call to all of them. But I don't know if that would be the right thing to do.

Reply:
Can't you just make it so the activating is in the part of the code that calls Activate()?

Reply:
nevermind its in the slime_extract